### PR TITLE
Enable tags for resources

### PR DESCRIPTION
**Pull request by sandeepgopi**
## what
implement a map(string) variable called tags and ensure this is implemented for:

- The 'aws_config_configuration_aggregator' resource with 'this' resource name
- The 'aws_config_aggregate_authorization' resource with 'child' resource name
- The ' aws_config_aggregate_authorization' resource with 'central' resource name

## why
When we began using the repo, we found that some of the resource are yet to be tag enabled. Hence when our internal projects were calling this repo, our custom tags were not being added in AWS. 

## references
* Closes https://github.com/cloudposse/terraform-aws-config/issues/50

### CODE REVIEW

**Comment by sandeepgopi** [comment]
Hi @nitrocode . Hope you are doing well. I see you are one of the authors for this repo. Please can my PR be reviewed and merged (It is to enable tags for some AWS resources), thank you :)

File: main.tf [hunk 1]
```diff
@@ -189,6 +189,7 @@ resource "aws_config_configuration_aggregator" "this" {
     account_ids = local.child_resource_collector_accounts
     all_regions = true
   }
+  tags = var.tags
```
Comment (nitrocode) [inline main.tf:192]:
```suggestion

  tags = module.this.tags
```

File: main.tf [hunk 2]
```diff
@@ -201,6 +202,7 @@ resource "aws_config_aggregate_authorization" "child" {
 
   account_id = var.central_resource_collector_account
   region     = var.global_resource_collector_region
+  tags       = var.tags
```
Comment (nitrocode) [inline main.tf:205]:
```suggestion

  tags = module.this.tags
```

**Comment by nitrocode** [comment]
Thank you @sandeepgopi for the contribution!

**Pull request merged**